src/app/views/invoices/invoice.controller.coffee

```diff
@@ -0,0 +1,29 @@
+@App.controller 'InvoiceController', ($stateParams, MnoeAdminConfig, MnoeInvoices) ->
+  'ngInject'
+  vm = this
+
+  vm.isPaymentEnabled = MnoeAdminConfig.isPaymentEnabled()
+  vm.invoice = {}
+  MnoeInvoices.get($stateParams.invoiceId).then(
+    (response) ->
+      vm.invoice = response.data.plain().invoice
+      vm.organization = response.data.plain().organization
+      vm.bills = response.data.plain().bills
+  ).finally(-> )
+
+  # The expected date is the 2nd of the month following the invoice period
+  vm.calculatePaymentDate = (endOfInvoiceDate) ->
```

Comment:
For the Date calculations logic, please use moment.
(cf. https://momentjs.com/)

Reply:
For the `ctrl.calculatePaymentDate(endOfInvoiceDate)` function.: please do not duplicate it. This code could be in a service.